pkg/phases/konfigmanager/deploy.go

```diff
@@ -0,0 +1,26 @@
+package konfigmanager
+
+import (
+	"github.com/flanksource/karina/pkg/constants"
+	"github.com/flanksource/karina/pkg/platform"
+)
+
+const (
+	Namespace = constants.PlatformSystem
+	Name      = "konfig-manager"
+)
+
+// Deploy deploys the konfig-manager into the platform-system namespace
+func Deploy(p *platform.Platform) error {
+	if p.KonfigManager.Version == "" {
+		p.KonfigManager.Version = "v0.2.1"
+	}
+	if p.KonfigManager.IsDisabled() {
```

Comment:
This will always return false, the version should not have defaults only the minimum components like nginx and cert-manager have defaults